modules/fishhook/1.0/config/default.yaml

```diff
@@ -0,0 +1,38 @@
+lcr-modules:
+    
+    fishhook:
+        inputs:
+            # Available wildcards: {seq_type}
+            master_maf: "__UPDATE__"
+            sample_sets: "__UPDATE__"
+
+        sample_set: ["__UPDATE__"]
+        seq_types: ["genome", "capture"]
+        include_non_coding: True
+        prepare_mafs: "{SCRIPTSDIR}/generate_smg_inputs/1.0/generate_smg_inputs.R"
+        src_dir: "{MODSDIR}/src/"
+
+        options:
+            tiles_size: 1000 #This is the size of tiles you want to split the genome into
+            coveriates: "_UPDATE_" #please supply a coveriate file in bed format, leave it blank if you want to run FishHook without coveriates
```

Comment:
We should leave this blank if the mode without covariates is preferred (or will be more often used), but thank you for documentation!

Reply:
Done